core/src/main/java/org/bitcoinj/script/ScriptError.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
Oh, please don't donate your copyright to Google...

You can simply use "Copyright by the original author or authors." here.